articles/multifactor-authentication/developer/step-up-with-acr.md

```diff
@@ -0,0 +1,76 @@
+---
```

Comment:
Agree in case of "logging in with a second factor", but I think that "logging in with a second factor" (which is usually handled by your framework) is not the same as "step-up" specially if you don't want the user to pass the whole authentication process as if it where re-logging in.